src/analyses/constant_propagator.cpp

```diff
@@ -167,8 +237,10 @@ bool constant_propagator_domaint::two_way_propagate_rec(
 #ifdef DEBUG
   std::cout << "two_way_propagate_rec: " << from_expr(ns,"",expr) << '\n';
 #endif
-  bool change = false;
 
+  bool change=false;
+
+#if 0
```

Comment:
It seems you are disabling the entire procedure. Would you mind adding a comment?